assets/src/adaptivity/rules-engine.ts

```diff
@@ -147,27 +151,38 @@ export const check = async (
   /* console.log('RE CHECK', { checkResult }); */
   let resultEvents: Event[] = [];
   const successEvents = checkResult.events.sort((a, b) => a.params?.order - b.params?.order);
+
   // if there are any correct in the success, get rid of the incorrect (defaultWrong most likely)
   const isCorrect = successEvents.some((evt) => evt.params?.correct === true);
-  if (isCorrect) {
-    resultEvents = successEvents.filter((evt) => evt.params?.correct === true);
+  //These are correct rules that do not have any condition so we need to make sure that the result does not have a wrong trap state otherwise it will
```

Comment:
hmm maybe we can split the rules stuff into a separate PR because I am not sure about this

Reply:
A separate PR created for value expression - https://github.com/Simon-Initiative/oli-torus/pull/1297